admin/sql/create_types.sql

```diff
@@ -7,3 +7,5 @@ CREATE TYPE user_relationship_enum AS ENUM('follow');
 CREATE TYPE recommendation_feedback_type_enum AS ENUM('like', 'love', 'dislike', 'hate', 'bad_recommendation');
 
 CREATE TYPE user_timeline_event_type_enum AS ENUM('recording_recommendation', 'notification');
+
+CREATE TYPE external_auth_service AS ENUM ('spotify', 'youtube');
```

Comment:
As far as I understand it, with Postgres > 9 it's quite easy to add an item to an enum at a later date.
Is that correct?

Reply:
Yes.